files/pitchpeek/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Jonathan Damon"
+github = "https://github.com/pitchpeek/"
```

Comment:
Thank you for that explanation. I put in the link to the full project, I hope that's correct, I've only put the minified version into the jsdelivr folder. Thank you.